crates/cairo-lint-core/tests/test_files/manual_unwrap_or_default/manual_unwrap_or_default

```diff
@@ -0,0 +1,385 @@
+//! > manual unwrap or default for if let with default
+
```

Comment:
add a test with a comment in either the match arm or the if else blocks

Reply:
Done.